README.md

```diff
@@ -107,21 +101,23 @@ The following flags can be passed to `cmake`:
 
 #### Servatrice Docker container
 
-A Dockerfile is provided to run Servatrice (the Cockatrice server) using [Docker](https://www.docker.com/what-docker).<br>
+Use [Docker](https://www.docker.com/what-docker) to run Servatrice (the Cockatrice server) using the Cockatrice Dockerfile.<br>
 
-You just need to create an image from the Dockerfile<br>
+First, create an image from the Dockerfile<br>
 `cd /path/to/Cockatrice-Repo/`
 `docker build -t servatrice .`<br>
 And then run it<br>
 `docker run -i -p 4747:4747/tcp -t servatrice:latest`<br>
 
->Please note that running this command will expose the TCP port 4747 of the docker container<br>
+>Note: Running this command exposes the TCP port 4747 of the docker container<br>
 to permit connections to the server.
 
-More infos on how to use Servatrice with Docker can be found in our [wiki](https://github.com/Cockatrice/Cockatrice/wiki/Setting-up-Servatrice#using-docker).
+Find more information on how to use Servatrice with Docker in our [wiki](https://github.com/Cockatrice/Cockatrice/wiki/Setting-up-Servatrice#using-docker).
+
+# ReadMe Style Guide and Documentation Process
 
+This ReadMe uses the [Google Developer Documentation Style Guide](https://developers.google.com/style/) to ensure consistent writing across the ReadMe.
```

Comment:
Why that huge headline?
The link isn't about `Documentation Process` too as far as I can tell.

Or do you want to add something about that topic here too?

<br>

We could also just use a bottom line at the very end instead a huge paragraph:
> This README uses the [Google Developer Documentation Style Guide](https://developers.google.com/style/) to ensure consistent writing(documentation?) across the project.

Same for CONTRIBUTING file or wiki main page!
Or even just a general hint here in readme that applies to all project.
`This project uses the ...`

Reply:
Merging it into an existing tab seems like a proper answer, I'll try to address this.